keps/sig-auth/3299-kms-v2-improvements/README.md

```diff
@@ -442,6 +443,27 @@ KMSv2REST-10      23.95Gi ± 0%       27.25Gi ± 0%  +13.77% (p=0.000 n=10)
 KMSv2REST-10       3.119M ± 0%       3.268M ± 1%  +4.78% (p=0.000 n=10)
 ```
 
+### Comparing Metrics
+
+Multiple [runs of e2e tests](https://gist.github.com/ritazh/528e627430621fffce41fd04983024a4) were performed to compare the impact of having KMS v2 encryption of all resources vs no encryption at all. The results are included below. 
+
+It shows that there is no significant increase in the following API server metrics: `apiserver_request_duration_seconds`, `apiserver_request_terminations_total`, `apiserver_request_aborts_total`.
+
+|                  | post*     | get*      | delete*   | list*     | request_terminations_total | request_aborts_total |
```

Comment:
Do we need the `request_terminations_total` and `request_aborts_total` columns?

Reply:
I added them for completeness since we called them out in the e2e metrics section. let me know what you prefer.

Reply:
I would just describe them in a sentence after the table.

Reply:
updated.